torch/_prims/executor.py

```diff
@@ -0,0 +1,13 @@
+from torch.fx import GraphModule
+from torch._prims.context import PrimContext
+
+
+def execute(ctx: PrimContext, *args, **kwargs):
+    """
+    Prototype ATen executor.
+
+    Just executes the context's graph.
+    """
+
+    gm = GraphModule({}, ctx.graph)
+    return gm.forward(*args, **kwargs)
```

Comment:
This doesn't seem to be tested anywhere?

Reply:
Correct - just used to support development of the prototype nvFuser Python API